plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/StatisticsNavigatorNodeRenderer.java

```diff
@@ -192,6 +197,17 @@ private void drawObjectDescription(@NotNull GC gc, @NotNull DBNDatabaseNode node
         }
     }
 
+    private void drawObjectChildrenCounter(@NotNull GC gc, @NotNull DBNDatabaseNode node, @NotNull Rectangle bounds) {
+        int childCount = 0;
+        try {
+            childCount = node.getChildren(new LocalCacheProgressMonitor(new VoidProgressMonitor())).length;
+        } catch (DBException e) {
+            // Ignore
```

Comment:
No need to show `(0)` if we can't for some reason read the children
```suggestion
            return;
```